test/parallel/test-stream-preprocess.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const fs = require('fs');
+const path = require('path');
+const rl = require('readline');
+
+const BOM = '\uFEFF';
+
+// get the data by a non-stream way to compare with the streamed data
```

Comment:
nit: Comments should be capitalized and punctuated; also, maybe `using` is better than `by`?